thunder/tests/opinfos.py

```diff
@@ -1646,18 +1646,25 @@ def celu_sample_generator(op, device, dtype, requires_grad):
 celu_opinfo = OpInfo(
     ltorch.celu,
     dtypes=(datatypes.floating,),
-    sample_input_generator=celu_sample_generator,
+    sample_input_generator=elementise_unary_with_alpha_generator,
     torch_reference=_elementwise_unary_torch(torch.celu),
-    test_directives=(
```

Comment:
This is cool! Why does adding `elu` improve the numeric consistency (with PyTorch) of `celu`?

Reply:
It's the change in test_grad.py that allows this.  Previously, the prologue had been omitted causing some unexpected (to me) things to happen surrounding the `alpha` parameter.

Reply:
Got it. It might make conceptual sense to separate the addition of the `elu` operation from the changes to the grad test (keeping this PR and creating a new one for the grad and `celu` changes).

Reply:
I await the approval and merging of [#1427](https://github.com/Lightning-AI/lightning-thunder/pull/1427) before proceeding here.

Reply:
#1427 has been merged and pulled into this PR.